src/util/myAxios.js

```diff
@@ -0,0 +1,86 @@
+const axios = require('axios');
+const stats = require('./stats');
+
+const fireLatencyStat = (startTime, statTags) => {
+  const integration = statTags.integration ? statTags.integration : '';
+  const type = statTags.type ? statTags.type : '';
+  stats.timing('outgoing_request_latency', startTime, {
+    type,
+    integration,
+  });
+};
+
+const send = async (options, statTags = {}) => {
+  const startTime = new Date();
+  let response;
+  try {
+    response = await axios(options);
+  } finally {
+    fireLatencyStat(startTime, statTags);
+  }
+  return response;
+};
+
+const get = async (url, options, statTags = {}) => {
```

Comment:
why do we need both network adapter and myAxios? can we combine both?

Reply:
Like I mentioned in the PR description, myAxios is temporary. Once all the network calls are migrated to use network adapter, myAxios can be removed safely.

Having this metric asap can be helpful. This can be a quick win.